ap_api/callbacks/customer/index.mdx

```diff
@@ -5,16 +5,19 @@ order: 0
 
 import {EndpointsTable} from "@site/src/components/EndpointsTable";
 
+Customers callback are designed to work with a customer data. When working with the Anchor Platform, business will store KYC data defined by [SEP-12].
+
+However, because the Anchor Platform does not store this data, it's required to fetch it from the business server. This set of endpoints allows to forward data stored on the business side to the wallet, in the SEP format.
```

Comment:
Lets start with the most important point here, which is that the anchor platform doesn't store KYC data, and that is why we need these endpoints.

```suggestion
The Anchor Platform does not store customer KYC data. Instead, it forwards this data to and fetches it from the business server. The customer callbacks are designed for this purpose. Currently, the requests and responses for these endpoints are almost identical to those defined in the SEP-12 KYC API specification.
```